src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -193,110 +246,93 @@ export function useSwapInputsController({
         }
       }
 
-      isFetching.value = false;
       quote.value = data;
-      runOnJS(updateQuoteStore)(data);
 
       if (!data || (data as QuoteError)?.error) {
         return;
       }
 
       if (inputAmount) {
```

Comment:
we now fetch asset prices along with the quote - we get price data for both input and output assets, but the quote will only return an inputAmount or outputAmount depending on what type of quote it was.

below, we are only updating the price for the input or output asset depending on the type of the quote (these are separate things, the prices for both assets can be updated)